src/platform_projects/openthread-efr32-soc.slcp

```diff
@@ -0,0 +1,38 @@
+project_name: openthread-efr32-soc
+label: ot-efr32
+package: OpenThread
+description: This is the platform project that will enable the ot-efr32 repo on GitHub to generate a CMake library for EFR32
+category: OpenThread Examples
+quality: production
+
+component:
+  - id: brd4001a
+  - id: ot_platform_abstraction_core
+  - id: ot_psa_crypto
+  - id: ot_thirdparty
+  - id: uartdrv_usart
+    instance:
+      - vcom
+  - id: rail_util_pti
+
+configuration:
+  - name: SL_BOARD_ENABLE_VCOM
+    value: 1
+  - name: SL_MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED
+    value: 1
+  - name: SL_MBEDTLS_KEY_EXCHANGE_PSK_ENABLED
+    value: 1
+  - name: UARTDRV_RESTRICT_ENERGY_MODE_TO_ALLOW_RECEPTION
+    value: 0
+    condition: [freertos, uartdrv_usart]
+  - name: SL_IOSTREAM_USART_VCOM_RESTRICT_ENERGY_MODE_TO_ALLOW_RECEPTION
+    value: 0
+    condition: [freertos, iostream_usart]
+  - name: SL_IOSTREAM_USART_VCOM_RX_BUFFER_SIZE
+    value: 128 # Increase from default 32
+    condition: [iostream_usart]
+  - name: SL_HEAP_SIZE
```

Comment:
Are the HEAP and STACK size really the same for all efr32?  I guess they maybe that makes sense.  Just checking.